cookbooks/aws-parallelcluster-test/recipes/test_neuron.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster-test
+# Recipe:: test_neuron
+#
+# Copyright:: 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+return if node['cluster']['base_os'] == 'centos7' || arm_instance? || node['cluster']['os'].end_with?("-custom")
+
+ruby_block "Verify Neuron repository is enabled" do
```

Comment:
I would move this check in tests done on build image rather than kitchen, since it doesn't depend on the instance type

Reply:
I'll do it in a separate patch.